CRUDRecipeEF.PL/Menus/RecipeMenu.cs

```diff
@@ -89,6 +94,95 @@ private async Task ExecuteMenuSelection(RecipeMenuOption option)
             }
         }
 
+        private async Task EditRecipe()
+        {
+            ConsoleHelper.ColorWrite("What Recipe would you like to edit: ");
+            var name = Console.ReadLine();
+
+            try
+            {
+                // this is super shit code, sorry
+                //TODO: Fix this mess
+                var recipe = await _recipeService.GetRecipeByName(name);
+                RecipeAddDTO recipeAdd = new RecipeAddDTO() { Name = recipe.Name };
```

Comment:
Automapper can fix this easily
1. We need a configuration inside the AutoMapperProfiles  CreateMap<from, to>
2. use the mapper like this: _mapper.Map<to>(fromObject)